clients/factory.go

```diff
@@ -183,6 +184,15 @@ func (f factory) SegmentsClient() (*segments.Client, error) {
 	return segments.NewClient(restClient), nil
 }
 
+// SLOClient creates and returns a new instance of slo.Client for interacting with the SLO API.
+func (f factory) SLOClient() (*slo.Client, error) {
```

Comment:
Follow up: Add to the existing tests in `factory_test.go`

Reply:
https://github.com/Dynatrace/dynatrace-configuration-as-code-core/pull/173